### ISSUE TITLE
I think I found a bug with the "Small Parts Table"

### ISSUE THREAD

**Issue by Bobdebilder** [AnonymousHacker1279/IWCompatibilityBridge#3]
Basically, whenever I craft any object with it, the game crashes 

I tried using copper the first time and cobalt the second, in order to see if it could have been related to modded ores (I have a copper overhaul installed, but the crash with your cobalt proved that the issue may be another)

Strangely enough, even if the game crashes, it still maintains the objects I crafted. But anytime I use that table, I have to launch the game from the start.

Could you see if its something in your code? I can send you my modlist, if you think it may be due to some incompatibility

Btw, great mod; can't wait to see what will you add in the future ;D

**Comment by AnonymousHacker1279** [comment]
I cannot reproduce any crashes with the small parts table myself; it's been working well on my end and other users have not reported any issues.

Could you upload your debug log? It will be located under `logs/debug.log` in your save folder. That will help me identify what is causing trouble. My guess is some incompatibility with other mods since I can't reproduce it. Hopefully the log will show if that's the case.

**Comment by Bobdebilder** [comment]
Sorry for the waiting: here I'm sending you two of the crashlogs that should mention the incident and a text document in which I pasted a part that seems to mention an error with the small parts table.
Do you want me to send my modlist as well?
Thank you again.
Let me know if you can read the files

[2022-07-18-1.log.gz](https://github.com/AnonymousHacker1279/ImmersiveWeapons/files/9150420/2022-07-18-1.log.gz)
[2022-07-19-2.log.gz](https://github.com/AnonymousHacker1279/ImmersiveWeapons/files/9150421/2022-07-19-2.log.gz)
[Small Parts table mention.txt](https://github.com/AnonymousHacker1279/ImmersiveWeapons/files/9150422/Small.Parts.table.mention.txt)

**Comment by AnonymousHacker1279** [comment]
This is actually not an issue with Immersive Weapons itself; the issue seems to reside in the compatibility bridge (IWCB). It seems like the Project MMO support feature in it is classloading non-existent classes. Basically, you don't have PMMO added but IWCB is trying to do stuff with it. 

I do find this odd because IWCB reported it *did not* initialize the plugin for PMMO:
```
[15:46:57] [Worker-Main-12/WARN]: Did not initialize plugin iwcompatbridge:pmmo_plugin, because it reported loading requirements were not met
```

I'm going to transfer this issue to the IWCB repository, and I'll start looking into it here soon.

**Comment by AnonymousHacker1279** [comment]
Confirmed to be a bug in the 1.19 release of IWCB as well (1.19-1.1.1). I'm going to work on fixing the 1.19 release first, then I will backport the fix to 1.18.2.

**Comment by AnonymousHacker1279** [comment]
Fixed in 1.18.2 with IWCB v1.0.3 and fixed in 1.19 with IWCB v1.1.2 and IW v1.18.1.

**Comment by Bobdebilder** [comment]
Just wanted to thank you for the fix, and I can't wait to see what else you are going to make
Have a great day!
